data/json/items/comestibles/bread.json

```diff
@@ -408,7 +408,7 @@
     "flags": [ "EDIBLE_FROZEN" ],
     "vitamins": [ [ "calcium", 2 ], [ "iron", 15 ] ],
     "stack_size": 20,
-    "fun": -1
+    "fun": -4
```

Comment:
-4 is a pretty hefty penalty for something that is tasteless.

Reply:
I never ate hardtack but I can imagine it sticks to your mouth and basically dries it out. Apparently its even that unpopular people of times past have written a song about it: https://en.wikipedia.org/wiki/Hard_Tack_Come_Again_No_More

Reply:
Hardtack is described as being so hard as to be unedible without some liquid and that soldiers on the march would occassionally be so desperately hungry that they would let it soak up the spit in their mouth to be soft enough to chew it so I wonder if -4 is enough.

Reply:
Apparenlty hardtack might be easy enough to make at home. There is only one way to find out: Make some hardtack for science!

Reply:
I think `-4` is a good number for consuming hardtack on its own but an ideal solution would add a recipe that allows the player to soak the hardtack in some water for 10-15 minutes to remove the Joy penalty (or return the value back to `-1`).